percebro/Dockerfile

```
@@ -107,11 +107,43 @@ RUN : \
     ; ldconfig \
     ;
 
-COPY install-vaapi.sh /tmp
-RUN : \
-    ; /tmp/install-vaapi.sh \
-    ; rm /tmp/install-vaapi.sh \
-    ;
+RUN if [ "${FORCE_VAAPI}" != "1" ]; then \
+        apt-get update && apt-get install -y pciutils; \
+        HAS_CARD=$(lspci | grep VGA | grep Intel); \
```

On system without Intel GPU the grep command returns 1 which fails build because -e option is enabled globally in the shell.

```suggestion
        HAS_CARD=$(lspci | grep VGA | grep Intel || true); \
```